asciidoctor-chart.gemspec

```diff
@@ -8,28 +10,32 @@ Gem::Specification.new do |s|
   s.name = 'asciidoctor-chart'
   s.version = Asciidoctor::Chart::VERSION
   s.summary = 'Adds a chart block and block macro to AsciiDoc'
-  s.description = 'A set of Asciidoctor extensions that add a chart block and block macro to AsciiDoc for including charts in your AsciiDoc document.'
+  s.description = "A set of Asciidoctor extensions that add a chart block and block macro to AsciiDoc
+for including charts in your AsciiDoc document."
   s.authors = ['Guillaume Grossetie']
   s.email = 'ggrossetie@gmail.com'
   s.homepage = 'https://asciidoctor.org'
   s.license = 'MIT'
-  # NOTE required ruby version is informational only; it's not enforced since it can't be overridden and can cause builds to break
-  #s.required_ruby_version = '>= 2.5.0'
+  # NOTE: required ruby version is informational only;
+  # it's not enforced since it can't be overridden and can cause builds to break
+  # s.required_ruby_version = '>= 2.7.0'
   s.metadata = {
     'bug_tracker_uri' => 'https://github.com/asciidoctor/asciidoctor-chart/issues',
-    #'changelog_uri' => 'https://github.com/asciidoctor/asciidoctor-chart/blob/master/CHANGELOG.adoc',
-    'mailing_list_uri' => 'http://discuss.asciidoctor.org',
-    'source_code_uri' => 'https://github.com/asciidoctor/asciidoctor-chart'
+    # 'changelog_uri' => 'https://github.com/asciidoctor/asciidoctor-chart/blob/master/CHANGELOG.adoc',
+    'community_chat_uri' => 'https://asciidoctor.zulipchat.com',
+    'source_code_uri' => 'https://github.com/asciidoctor/asciidoctor-chart',
+    'rubygems_mfa_required' => 'true'
```

Comment:
According to https://github.com/rubygems/rubygems.org/issues/2500#issuecomment-755080919 you should enable MFA on https://rubygems.org/ and use "UI and gem signin".

In theory, it would be possible to publish using an API key (from CI/CD) even though MFA is enabled (and required).